torchmetrics/classification/precision_recall.py

```diff
@@ -52,10 +52,8 @@ class Precision(StatScores):
             - ``'samples'``: Calculate the metric for each sample, and average the metrics
               across samples (with equal weights for each sample).
 
-            Note that what is considered a sample in the multi-dimensional multi-class case
-            depends on the value of ``mdmc_average``.
-        multilabel:
-            .. warning :: This parameter is deprecated and has no effect. Will be removed in v1.4.0.
+            .. note:: What is considered a sample in the multi-dimensional multi-class case
+                depends on the value of ``mdmc_average``.
```

Comment:
```suggestion
            .. note:: A sample considered in the multi-dimensional multi-class case
                depends on the value of ``mdmc_average``.
```
Similar suggestion to other changes.